src/tests/stub_debug.cc

```diff
@@ -14,9 +14,9 @@
  * not abort the unit test.
  */
 #include "squid.h"
-#include "Debug.h"
+#include "debug/Stream.h"
```

Comment:
Now that the object being stub'ed is a library there are a few changes relevant to this file to bring it inline with the other library stubs.

 * rename to stub_libdebug.cc and update related references to the filename, STUBS.list primarily.
 * all .h files in the library have an include statement below the STUB_API definition (not in the top #include list),
 * each library .h include is followed by the block of stubs for *all* its public API symbols.

Reply:
We should not do that in this PR because the API exposed by libdebug may change quite a bit in the followup PR(s). If you insist on doing these changes now, I will probably waste time on stubbing APIs that will be changed or removed very soon. Let's revisit after the API-related changes are done in followup PRs. I will volunteer to update tests/stub_debug.cc at that time.

There is even a possibility that we will not need to stub libdebug at all after this PR series is complete.

Reply:
Okay. I can accept it as a followup.